appveyor.yml

```diff
@@ -11,27 +11,30 @@ image:
   - Visual Studio 2013
 
 environment:
+  DEBUG_BATCH: "1"
   matrix:
   - TEST_ALL_EXTRAS:
 
 build_script:
-- ps: |
-    & .\bootstrap.bat
-    $BootstrapExitCode = $LastExitCode
-    if (Test-Path bootstrap.log)
-    {
-        Push-AppveyorArtifact bootstrap.log
-    }
-    if ($BootstrapExitCode -ne 0) { $host.SetShouldExit($BootstrapExitCode) }
+- cmd: |
+    set > set.log
+
+    bootstrap.bat
 
 test_script:
 - ps: |
     cd test
-    & python test_all.py msvc $env:TEST_ALL_EXTRAS --preserve
-    $TestExitCode = $LastExitCode
-    if (Test-Path failed_test)
-    {
-        & 7z a failed_test.zip failed_test\*
-        Push-AppveyorArtifact failed_test.zip
-    }
-    if ($TestExitCode -ne 0) { $host.SetShouldExit($TestExitCode) }
+    
+    python test_all.py msvc $env:TEST_ALL_EXTRAS --preserve
```

Comment:
Not sure why? This works (as it should).

P.S. __&__ operator does not parse variables.
From the docs:

__&__ Call operator
==
Run a command, script, or script block.
The call operator, also known as the "invocation operator," lets you run commands that are stored in variables and represented by strings.
__Because the call operator does not parse the command, it cannot interpret command parameters.__

```
C:\PS> $c = "get-executionpolicy"
C:\PS> $c
get-executionpolicy
C:\PS> & $c
AllSigned
```